### PR TITLE
Notify all dojo members that a new event has been published and it's ready for booking

### PR DESCRIPTION
**Pull request by ckiss**
ref https://github.com/CoderDojo/community-platform/issues/864

### CODE REVIEW

**Comment by dberesford** [comment]
LGTM

File: web/public/js/controllers/manage-dojo-events-controller.js [hunk 1]
```diff
@@ -66,19 +66,26 @@
...
+      }, function (value) {
+        return value === '' || _.isNull(value) || _.isUndefined(value)
+      });
 
       var loadPageData = tableUtils.loadPage(resetFlag, $scope.pagination.itemsPerPage, $scope.pagination.pageNo, query);
       $scope.pagination.pageNo = loadPageData.pageNo;
       $scope.events = [];
-      cdEventsService.search({dojoId: $scope.dojoId, limit$: $scope.pagination.itemsPerPage, skip$: loadPageData.skip, sort$: $scope.sort}).then(function (result) {
+      cdEventsService.search({
+        dojoId: $scope.dojoId,
```
Comment (Wardormeur) [inline web/public/js/controllers/manage-dojo-events-controller.js]: Thanks for that :+1:

File: web/public/js/controllers/manage-dojo-events-controller.js [hunk 2]
```diff
@@ -157,10 +164,29 @@
     $scope.loadPage($scope.filter, true);
 
     $scope.getSortClass = utilsService.getSortClass;
+
+    $scope.showNotifyAllMembers = function () {
+      var notifyAllMembersModal = $modal.open({
+        animation: true,
```
Comment (Wardormeur) [inline web/public/js/controllers/manage-dojo-events-controller.js]: Default is true :)

**Comment by Wardormeur** [comment]
LGTM, thanks :+1:

**Comment by Wardormeur** [comment]
Will probably conflict after https://github.com/denniskenny/cp-zen-platform/commit/b774a7f8f4d28967420a1040435dfbb72b76de19 is merged, sorry about that :|

**Pull request merged**